### PR TITLE
ray 2.7.1 in readme (plus syncing quickstart with docs)

### PR DESCRIPTION
**Pull request by mateuszsrebrny**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,45 +33,42 @@ If you have any questions, comments, insights, praises, or doubts about these do
...
 
 ## Install software
 
+**Note:** We recommend creating a new directory and a clean Python virtual environment before you proceed. This avoids cluttering your system installation with unnecessary packages.
+
 The first step is installing Ray on Golem. It will install Ray as a dependency.
 
 ```bash
-# install ray-on-golem and ray (recommended within a clean venv)
+# install ray-on-golem & ray (recommended within a clean virtual environment)
```
Comment (shadeofblue) [inline README.md:63]: what's wrong with `and` ?

Comment (mateuszsrebrny) [reply README.md:63]: nothing, is it for consistency with changes in the docs, we had a couple lines like that there, had to pick one

**Review by shadeofblue — APPROVED** [review]
why the `&` instead of just `and` ?

**Comment by mateuszsrebrny** [comment]
> why the `&` instead of just `and` ?

dunno, just picked one on random - does it make a difference?

**Comment by shadeofblue** [comment]
> > why the `&` instead of just `and` ?
> 
> dunno, just picked one on random - does it make a difference?

well, it's a colloquialism ... and you don't use `&` in normal sentences...

**Pull request merged**